01week/datatypes.js

```diff
@@ -0,0 +1,115 @@
+'use strict'
+
+// logs current date
+
+let displayDate = function() {
+	let date = new Date();
+	console.log(date);
+	console.log(typeof(date));
+}
+
+displayDate();
+
+
+// convert number to string
+
+let convertNumber = function() {
+	let newNumber = 5;
+	toString(newNumber);
```

Comment:
This is not correct. see https://www.w3schools.com/jsref/jsref_tostring_number.asp